docs/bigquery/snippets.py

```diff
@@ -2155,24 +2153,28 @@ def test_undelete_table(client, to_delete):
 
     table_ref = client.dataset(dataset_id).table(table_id)
 
-    # Record the current time in milliseconds. We'll use this as the snapshot
-    # time for recovering the table.
-    snapshot_time = int(time.time() * 1000)
+    # TODO(Developer):  Choose an appropriate snapshot point as epoch
```

Comment:
Make the whole comment part of the TODO (no newline). The separation in the comment makes the TODO less clear because it looks like a different section.